### DISCUSSION TITLE
Additional integration mod function

### DISCUSSION THREAD

**Discussion by ssiwm** [gameserverapp/Platform#566] [Suggestions]
While waiting for the launch of the official shop mod, I came across kavan87's solution, namely Kavan's Shop & Missions.

https://www.curseforge.com/ark-survival-ascended/mods/kavans-shop-missions/screenshots

 The mod has a very useful function - it automatically searches game and mod files(I think so, I don't really know what's going on in the background, but after adding the mod, Dino and its items automatically appear on the list) and adds Dino and items to the list to which the admin has access.  
The advantage is that I don't need to know BP patch to add something to the store.  I log in as admin in the game and have access to the list from which I can add Dino and items to the store.

I have no idea whether such a function is possible in the newly developed application, but many admins would certainly be happy with such a convenience.

**Comment by mvaessen** [comment]
Hi, thanks for your suggestion. While I understand the advantage for admins, is this very labor intensive to maintain.
If that mod has everything you want, I recommend using the mod 👍
